doc/userguide/rules/base64-keywords.rst

```diff
@@ -0,0 +1,50 @@
+Base64 keywords
+===============
+
+Suricata supports decoding base64 encoded data from buffers and matching on the decoded data.
+
+This is achieved by using two keywords, ``base64_decode`` and ``base64_data``. Both keywords must be used in order to generate an alert.
+
+These keywords are compatible with their Snort counterparts.
+
+base64_decode
+-------------
+
+Decodes base64_data from a buffer and makes it available for the base64_data function.
+
+Syntax::
+
+    base64_decode:bytes <value>, offset <value>, relative;
+
+The ``bytes`` option specifies how many bytes that should be decoded and made available for base64_data.
+The decoding will stop at the end of the buffer.
+
+The ``offset`` option specifies how many bytes to skip before the decoding should start. Bytes are skipped relative to the start of the payload buffer if relative is not set.
+
+The ``relative`` option makes the decoding start relative to the previous content match. Default behavior is to start at the beginning  of the buffer. This option makes ``offset`` skip bytes relative to the previous match.
```

Comment:
between beginning and of the buffer are double spaces.
It might also be better to have a line break.

Reply:
I'm sorry, but I don't fully understand the first sentence.
I'll line breaks for my next pull request.

Reply:
between "Default behavior is to start at the beginning" and "of the buffer." are two spaces and not just one.